handler.go

```diff
@@ -252,7 +318,9 @@ func (h FilteredHttpRequestHandler) Handle(req *http.Request, metrics Metrics) (
 		_, ok := h.allowedOrigins[req.Host]
 		if !ok {
 			metrics.Fire(metricsResultTargetRequestForbidden)
-			return nil, TargetForbiddenError
+			// to allow clients to fix improper third party urls usage (e.g. to change URLs from our direct s3 refs to CDN)
+			log.Printf("TargetForbiddenError: %s, %s", req.Host, req.URL)
```

Comment:
I'd prefer to see this somehow gated by a configuration variable. It shouldn't always log unless the user wants to see these log messages, for example.

Reply:
done under verbose